srtcore/core.cpp

```diff
@@ -6497,7 +6497,7 @@ int srt::CUDT::sndDropTooLate()
 
     // If some packets were dropped update stats, socket state, loss list and the parent group if any.
     enterCS(m_StatsLock);
-    m_stats.sndr.dropped.count(dbytes);;
+    m_stats.sndr.dropped.count(stats::BytesPackets((uint64_t) dbytes, (uint32_t) dpkts));
```

Comment:
Now `BytesPacketsCount` instead of `BytesPackets` 😮 
```suggestion
    m_stats.sndr.dropped.count(stats::BytesPacketsCount((uint64_t) dbytes, (uint32_t) dpkts));
```

Reply:
Likely a merge break.